src/window.ts

```diff
@@ -98,6 +98,13 @@ export class Window extends Adw.ApplicationWindow {
     this.default_width = note.width;
     this.default_height = note.height;
 
+    this.note.bind_property(
+      "title",
+      this,
+      "title",
+      GObject.BindingFlags.SYNC_CREATE
```

Comment:
I think a window having an empty title is problematic. Could you modify the binding so that it sets the title to `_("Sticky Note")` if it's empty?

Reply:
I set the title to `Untitled Note` if it's empty